src/x11/display.rs

```diff
@@ -19,17 +20,32 @@ pub struct Rect {
 }
 
 impl Display {
+    ///
+    /// # Safety
     pub unsafe fn new(
```

Comment:
Not sure exactly what an appropriate "safety" doc comment would be here but clippy complains without it so I stubed it in.